codalab/lib/bundle_cli.py

```diff
@@ -171,7 +171,7 @@ def print_help(self, out_file=None):
     def error(self, message):
         # Adapted from original:
         # https://hg.python.org/cpython/file/2.7/Lib/argparse.py
-        if len(sys.argv) == 1 or (len(sys.argv) == 2 and sys.argv[1] in ['--help', '-h']):
+        if len(sys.argv) == 1 or len(sys.argv) == 2 and sys.argv[1] in ['--help', '-h']:
```

Comment:
I think you meant to switch this back to what it was before:

```suggestion
        if len(sys.argv) == 1 or (len(sys.argv) == 2 and sys.argv[1] in ['--help', '-h']):
```